lib/api/controllers/documentController.js

```diff
@@ -72,11 +73,14 @@ function DocumentController (kuzzle) {
     assertBodyAttributeType(request, 'ids', 'array');
     assertHasIndexAndCollection(request);
 
+    if (request.input.body.ids.length > kuzzle.config.server.maxMultiActionsCount) {
```

Comment:
First, I would say mGet is faster than search to get a list of documents by their ids.
Second, I don't really see the point of associating the `maxMultiActionsCount` with `mGet`; I thought the purpose of the settings was to avoid a `m*` to stuck Kuzzle completely because it was not going through the overload system properly.
At this point we should limit the size parameter of search requests as well...